tools/tcpaccept.py

```diff
@@ -21,21 +21,27 @@
 from struct import pack
 import argparse
 from bcc.utils import printb
+from time import strftime
 
 # arguments
 examples = """examples:
     ./tcpaccept           # trace all TCP accept()s
     ./tcpaccept -t        # include timestamps
+    ./tcpaccept -P 80     # only trace port 80
     ./tcpaccept -p 181    # only trace PID 181
 """
 parser = argparse.ArgumentParser(
     description="Trace TCP accepts",
     formatter_class=argparse.RawDescriptionHelpFormatter,
     epilog=examples)
+parser.add_argument("-T", "--time", action="store_true",
+    help="include time column on output (HH:MM:SS)")
 parser.add_argument("-t", "--timestamp", action="store_true",
```

Comment:
Should we make -t and -T exclusive to each other? It probably does make sense to have both in outputs?

Reply:
With `tcplife` we can:
    `./tcplife -stT      # csv output, with times & timestamps`
So I guess it is ok to make -t and -T non-exclusive.